discord/http.py

```diff
@@ -220,16 +220,16 @@ def recreate(self):
             # We've run out of retries, raise.
             raise HTTPException(r, data)
 
-    async def get_attachment(self, url):
+    async def get_from_cdn(self, url):
         async with self._session.get(url) as resp:
             if resp.status == 200:
                 return await resp.read()
             elif resp.status == 404:
-                raise NotFound(resp, 'attachment not found')
+                raise NotFound(resp, 'media not found')
```

Comment:
```suggestion
                raise NotFound(resp, 'asset not found')
```

The class name is `Asset` not `Media`. Same for the rest of the strings.